pages/index.tsx

```diff
@@ -10,6 +14,137 @@ const Home = () => {
       </Head>
       <Hero />
       <WhyKedro />
+      <section className={style.features}>
+        <h3 className={style.sectionTitle}>Features</h3>
+        <FeatureDetails
+          buttonLink="https://demo.kedro.org/"
+          buttonText="Explore Live Demo"
+          iframeLink="https://demo.kedro.org/"
+          subtitle={
+            <>
+              <a
+                href="https://github.com/kedro-org/kedro-viz"
+                rel="noopener noreferrer"
+                target="_blank"
+              >
+                Kedro&#39;s pipeline visualisation plugin
+              </a>{' '}
+              shows a blueprint of your developing data and machine-learning
+              workflows, provides data lineage, keeps track of machine-learning
+              experiments and makes it easier to collaborate with business
+              stakeholders.
+            </>
+          }
+          title="Pipeline visualisation"
+        />
+        <FeatureDetails
+          altText="Placeholder"
+          assetPosition="right"
+          buttonLink="https://kedro.readthedocs.io/en/stable/02_get_started/05_example_project.html#project-directory-structure"
+          buttonText="Learn more"
+          imageSrc={placeholder}
```

Comment:
This image is provided on Figma and can be slotted directly
<img width="1156" alt="image" src="https://user-images.githubusercontent.com/27775658/155740213-bd20f93f-772b-45b5-801e-9e94c83c9d62.png">

Reply:
Added.